### PR TITLE
initial commit with new branch

### PR DESCRIPTION
**Pull request by mayra-tellez**

### CODE REVIEW

File: aboutPage/about.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <a href="../contactPage/contact.html">Contact</a>
+      </nav>
+    </header>
+    <main class="content">
+      <h1>Trims Started</h1>
+      <img id="shopPhoto" src="../images/barberShop.jpg" alt="barber shop with brick wall and empty chairs">
+      <article>
+        <p>
+        <h3>Hair Cuts for Everyone</h3>
+        Our website templates are created with inspiration, checked for quality and originality and meticulously sliced and coded. What’s more, they’re absolutely free! You can do a lot with them. You can modify them. You can use them to design websites for clients, so long as you agree with the Terms of Use. You can even remove all our links if you want to.
```
Comment (juan-rod) [inline aboutPage/about.html:35]: we usually don't nest h1/h2/h3/h4/h5/h6 tags inside a p tag. H3 is a heading tag so it labels a title of something. p tag is a paragraph which will go under a title. no need to change it now. might be too messy to move things around but just know for the future!

**Review by juan-rod — APPROVED** [review]
really great work! and you made the site look way better! there's not much to critique! you have all the pages. it's responsive. What questions do you have? Let me know if you want to go over something specific either in class or outside of it. 

Merge when ready.
after you merge, open up your terminal/gitbash. go to this folder and switch to your main branch. type the command "git pull origin main". you should see the new merged files.

File: aboutPage/about.html (hunk 1 above)
Comment (mayra-tellez) [reply aboutPage/about.html:35]: ahh silly me. got it!

**Comment by mayra-tellez** [comment]
awesomeee, thank you so much, Juan!!! Would I ask questions here or directly to you? Just because I don't know what happens after a merge, but my main question is about the contact form. Should we be able to actually capture data? I created a results.html page and entered this line of code for the form: <form action="results.html" method="GET">, but I'm not actually sure what happens with this or where the data is sent. Hope that question made sense!

**Comment by mayra-tellez** [comment]
`<form action="results.html" method="GET">`

**Pull request merged**